### ISSUE TITLE
Forward/Backward button moves files?

### ISSUE THREAD

**Issue by Rajdave69** [aleksey-hoffman/sigma-file-manager#257]
### Is there an existing issue for this?

- [X] I have searched the opened issues

### Problem description

If you have a mouse with forward/backward buttons and if you hold either one of the buttons on top of a file it starts to move it by dragging. (Look at image)

![image](https://github.com/aleksey-hoffman/sigma-file-manager/assets/86198810/9fb2556c-6816-4a16-9ab6-462f9af04cd0)

Not sure if this is intended or not, and if both buttons are supposed to be used for dragging
If this is intended, perhaps a different function for each of the buttons? forward for copy and backward for move maybe?

If this is not intended a good use for the buttons (in addition to functioning as the back button and forward button) would maybe be copy/paste or copy/cut? idk

To clarify, I'm talking about these buttons - 
![image](https://github.com/aleksey-hoffman/sigma-file-manager/assets/86198810/d074c95d-72d4-494e-a059-af5095a01f8f)

4 and 6

### Steps to reproduce

1. hold the forward/backward mouse button on top of a file or folder

### Environment

- OS: Windows 11 build 22622.601
- App version: latest (not sure where to find version)

**Comment by Wateraga** [comment]
Yes I have this issue too.. very annoying when trying to be quick about forward/back and means you have to be careful where you click for simple forward/back.

Also as an improvement I suggest adding customisation functionality for the mouse 3/4/5 buttons, as it is the 3/4 buttons are actually 4/5 and you can't cstomise mouse 3(middle click)

**Comment by Wateraga** [comment]
Also have this issue, makes the software borderline unusable as a replacement for Explorer sue to accidental drag moves when using the Forward/Back buttons.

Would reallyt like a fix for this as I would love to completely replace Explorer with Sigma.

Love sigma in general and appreciate the effort.
